woocommerce/woocommerce-gutenberg-products-block
# Remove/snackbar filters

This PR reverts #4222 and also #4313 

`wc-blocks-checkout` was being made a dependency of "All Products", "Filters" and "Reviews" blocks because of the `Label` and `Button` component being moved to `@woocommerce/blocks-checkout` and then consumed by these non-checkout blocks.

Also, `All Products` was affected because it used the `StoreNoticeProvider` which had a dependency on `@woocommerce/blocks-checkout` because of the `__experimentalApplyCheckoutFilter` used to filter snackbar notices. All Products doesn't use the snackbar notices, but the dependency still exists because store notices and snackbar notices are tightly coupled.

Fixes #4391 

### How to test the changes in this Pull Request:

1. Ensure that the `wc-blocks-checkout` is not listed as a dependency in any of the `[name].asset.php` files except the cart and checkout ones.
2. Add items to your cart and then go to the Cart block. Apply a coupon.
3. Ensure the snackbar notice is shown in the bottom left of the screen.
4. Remove the coupon, ensure the snackbar notice is shown.



### Changelog

> Remove the ability to filter snackbar notices
